go.mod

```diff
@@ -24,6 +24,7 @@ require (
 replace (
 	github.com/containers/buildah => github.com/containers/buildah v1.23.1
 	github.com/containers/common => github.com/containers/common v0.46.1-0.20211026130826-7abfd453c86f
+	github.com/jakub-dzon/k4e-operator => github.com/ydayagi/k4e-operator v0.0.0-20211208101641-8b430f6c0f5b
```

Comment:
this will be removed after https://github.com/jakub-dzon/k4e-operator/pull/47 is merged

Reply:
and then line 13 must be updated